### PR TITLE
chore: Implement new workflow for updating of the translations from Phrase.com

### PR DESCRIPTION
**Pull request by borisprimer**

### CODE REVIEW

File: .github/workflows/manual-translation-update.yml [hunk 1]
```diff
@@ -12,33 +11,40 @@ on:
...
+          PHRASE_ACCESS_KEY: ${{ secrets.PHRASE_ACCESS_KEY }}
+          PHRASE_PROJECT_ID: ${{ secrets.PHRASE_PROJECT_ID }}
 
       - name: Download Translations from Phrase
-        run: phrase pull --project-id ${{ secrets.PHRASE_PROJECT_ID }} --branch ${{ github.event.inputs.branch }}
+        run: phrase pull
 
       - name: Commit and Push Changes
         run: |
           git config --global user.email "action@github.com"
```
Comment (NQuinn27) [inline .github/workflows/manual-translation-update.yml:42]: We dont need the --global here or with user.name `git config user.email action@github.com` will do - also no need for the quote marks (although is probably fine with them).

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,2 @@
-*.strings diff
+*.strings text
+*.strings diff
```
Comment (borisprimer) [inline .gitattributes:2]: @NQuinn27 @jnewc @semirp what is this red circle with a dash?

Comment (NQuinn27) [reply .gitattributes:2]: It's applying the effect to all files of type string - `*.string`

https://git-scm.com/docs/gitattributes

**Review by NQuinn27 — APPROVED** [review]
Looks good

File: .gitattributes (hunk 2 above)
Comment (borisprimer) [reply .gitattributes:2]: I was asking about the UI here in GitHub, if you open https://github.com/primer-io/primer-sdk-ios/pull/918/files you will see it.

Comment (borisprimer) [reply .gitattributes:2]: Screenshot 2024-06-28 at 14 52 50

**Pull request merged**